### PR TITLE
Initial version of package

### PR DESCRIPTION
**Pull request by neveritt**

### CODE REVIEW

File: src/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  fast_finish: true
+  allow_failures:
+    - julia: nightly
+
+notifications:
+  email: false
+
+script:
+  - julia -e 'Pkg.clone(pwd())'
+  - julia --check-bounds=yes --inline=no -e 'Pkg.test("{[{package}]}",
```
Comment (aytekinar) [inline src/.travis.yml]: You have forgot to change the `{[{package}]}` placeholder.

File: src/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    coverage=true)'
+
+after_success:
+  - julia -e 'cd(Pkg.dir("{[{package}]}")); Pkg.add("Coverage"); using Coverage;
```
Comment (aytekinar) [inline src/.travis.yml]: `{[{package}]}` place holder.

Well, if you enable CI through Travis for this repo, you will notice this yourself. Maybe it's better to just enable it and see how the build goes.

File: src/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    Coveralls.submit(process_folder()); Codecov.submit(process_folder())'
```
Comment (aytekinar) [inline src/.travis.yml]: we need to put a _new line_, here, apparently.

**Review by aytekinar — CHANGES_REQUESTED** [review]
Well, if you don't address the `inline` comments, it will not build anyways --- that's why I have _requested_ changes. Otherwise, I did not check the code itself, but seems neat.

Some notes:
1.  I am not sure if you will squash everything before the first tag, but if not, please change the `stupid stuff` commit message, at least ;)
2.  Later on, I will also try to change the code in `ControlCore`, for example, to use the Unicode (i.e., actual Greek) letters of `alpha`, `beta`, etc. Maybe you can find and replace these `lambda`s with `\lambda`s. This is just a minor thing, obviously :)

File: src/.travis.yml (hunk 2 above)
Comment (neveritt) [reply src/.travis.yml]: good thinking.

File: src/.travis.yml (hunk 3 above)
Comment (neveritt) [reply src/.travis.yml]: apparently

**Review by aytekinar — APPROVED** [review]

**Pull request merged**